ffig/generators/python.py

```diff
@@ -0,0 +1,28 @@
+# Generator module for Python 2 and 3.
+
+import generators
+import os
+
+
+def generator(module_name, binding, api_classes, env, output_dir):
+    module_dir = os.path.join(output_dir, module_name)
+    if not os.path.exists(module_dir):
+        os.makedirs(module_dir)
+
+    with open(os.path.join(module_dir, '__init__.py'), 'w') as o:
+        o.write(
+            """import sys
+if sys.version_info[0] == 3:  
+    from .interop_py3 import *
+else:
+    from .interop_py2 import *
+""")
+    for o in [os.path.join(module_dir, x) for x in ['interop_py2.py', 'interop_py3.py']]:
+        generators.generate_single_output_file(
+            module_name, 'py.tmpl', api_classes, env, o)
+
+    return [os.path.join(module_dir, x) for x in ['__init__.py', 'interop_py2.py', 'interop_py3.py']]
+
+
```

Comment:
I dislike unnecessary blank lines, but if that's the official Python way, then fair enough.

We should be consistent in our use of autopep8 / clang-format / etc. on the FFIG code itself. If we're going to use it, we should probably add a section in the Travis build to run autopep8 (and clang-format on the C++ files) and fail the build if `git status` shows differences after running it. I'd stop short of automatically committing the reformatted version, but this should be enough to remind the committer that they need to run those tools.